plugins/huawei/mm-broadband-bearer-huawei.c

```diff
@@ -392,7 +392,7 @@ connect_3gpp_context_step (Connect3gppContext *ctx)
 
         if (g_strcmp0 (mm_iface_modem_get_model (MM_IFACE_MODEM (ctx->modem)), "E8278") == 0) {
             mm_dbg ("Skip NDISDUP and NDISSTATQRY for E8278");
-            ctx->step = CONNECT_3GPP_CONTEXT_STEP_LAST;
+            ctx->step = CONNECT_3GPP_CONTEXT_STEP_IP_CONFIG;
             connect_3gpp_context_step (ctx);
             return;
         }
```

Comment:
To protect ourselves from this type of change (new states being added between `NDISSTATQRY` and `IP_CONFIG`), and ease future rebases, what about skipping each individual step? Something like this on the beginning of each case block:
```
if (g_strcmp0 (mm_iface_modem_get_model (MM_IFACE_MODEM (ctx->modem)), "E8278") == 0) {                                                                                                                                             
    mm_dbg ("Skip NDISDUP for E8278");
    ctx->step++;
    /* Fall down to the next step */
    break;  
}
```